src/test/java/com/google/alpollo/DataBaseTest.java

```diff
@@ -0,0 +1,116 @@
+package com.google.alpollo;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.HashMap;
+import com.google.alpollo.model.Song;
+import com.google.alpollo.model.SongCounter;
+import com.google.alpollo.model.AnalysisInfo;
+import com.google.alpollo.SongDataBase;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.googlecode.objectify.ObjectifyFactory;
+import com.googlecode.objectify.ObjectifyService;
+import com.googlecode.objectify.cache.AsyncCacheFilter;
+import com.googlecode.objectify.util.Closeable;
+import org.junit.BeforeClass;
+import org.junit.Before;
+import org.junit.After;
+
+@RunWith(JUnit4.class)
+public final class DataBaseTest {
+  private final LocalServiceTestHelper helper = new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  private Closeable session;
+
+  private static final Song SONG = new Song("Drake", "Toosie Slide");
+  private static final int TEN = 10;
```

Comment:
Explain what `TEN` means and why we have it, please (add a comment, for example)